Dockerfile

```diff
@@ -28,31 +28,31 @@ WORKDIR /app
 
 COPY Gemfile Gemfile.lock package.json yarn.lock .ruby-version ./
 
-RUN apk -U upgrade && \
-    apk add --update --no-cache --virtual .gem-installdeps $BUILD_DEPS && \
-    gem update --system && \
-    find / -wholename '*default/bundler-*.gemspec' -delete && \
-    rm -rf /usr/local/bin/bundle && \
-    gem install bundler -v 2.3.9 && \
-    bundler -v && \
-    bundle config set no-cache 'true' && \
-    bundle config set no-binstubs 'true' && \
-    bundle --retry=5 --jobs=4 --without=development test && \
-    yarn install --check-files --production && \
-    apk del .gem-installdeps && \
-    rm -rf /usr/local/bundle/cache && \
-    find /usr/local/bundle/gems -name "*.c" -delete && \
-    find /usr/local/bundle/gems -name "*.h" -delete && \
-    find /usr/local/bundle/gems -name "*.o" -delete
+RUN apk -U upgrade 
+RUN apk add --update --no-cache --virtual .gem-installdeps $BUILD_DEPS
+RUN gem update --system
+RUN find / -wholename '*default/bundler-*.gemspec' -delete
+RUN rm -rf /usr/local/bin/bundle
+RUN gem install bundler -v 2.3.9
+RUN bundler -v
+RUN bundle config set no-cache 'true'
+RUN bundle config set no-binstubs 'true'
+RUN bundle --retry=5 --jobs=4 --without=development test
+RUN yarn install --check-files --production
+RUN apk del .gem-installdeps
+RUN rm -rf /usr/local/bundle/cache
+RUN find /usr/local/bundle/gems -name "*.c" -delete
+RUN find /usr/local/bundle/gems -name "*.h" -delete
+RUN find /usr/local/bundle/gems -name "*.o" -delete
```

Comment:
you'd want to run all of those in one in single comment, hence the previous `&& \` in order for docker to cache the layer adequately.